### DISCUSSION TITLE
How to change display flash message time?

### DISCUSSION THREAD

**Discussion by PeopleInside** [symfony/symfony#44324] [Q&A] [answered]
Hi, I'm using an open Source Help Desk project who use Symfony.

It use flash confirmation and error message from Symfony but in case of !Warning the time is too short because the Warning message is added below another successfully message, text of the confirmation and warning message are long so time is not enough to read the full message.
 
I asked to the Open Source team to change that and they said that timing is managed by Symfony.
What can i do to have a better longer Warning flash message timing?

Thanks

![Animation](https://user-images.githubusercontent.com/5006150/143830413-e65a2719-6173-454e-9dbf-83eeae35da1a.gif)

**Comment by youssefsaoubou** [comment, marked as answer]
You can use https://php-flasher.github.io/  to custom your flasher

**Reply by PeopleInside** [reply]
> You can use https://php-flasher.github.io/ to custom your flasher

I have to suggest to the team of the Open Source to use this php flasher or there is something as end user I can do on my server, considering I'm not a developer, to resolve the short time push messages showed?

Thanks for your useful answer.
